src/groups/bmq/bmqp/bmqp_protocol.cpp

```diff
@@ -454,7 +454,7 @@ bool PutHeaderFlags::fromAscii(PutHeaderFlags::Enum*    out,
 #define CHECKVALUE(M)                                                         \
     if (bdlb::String::areEqualCaseless(toAscii(PutHeaderFlags::e_##M),        \
                                        str.data(),                            \
-                                       str.length())) {                       \
+                                       static_cast<int>(str.length()))) {     \
```

Comment:
These should be relatively straightforward fixes similar to previous 'mwcio_channel.cpp' fixes in #216 . We merely acknowledge that converting ‘BloombergLP::bslstl::StringRefImp<char>::size_type’ {aka ‘long unsigned int’} to ‘int’ is fine and ‘static_cast‘ to reduce unnecessary build logs.